test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -708,7 +708,7 @@ public HubConnectionTests(ServerFixture<Startup> serverFixture, ITestOutputHelpe
             {
                 var httpResponse = await new HttpClient().GetAsync(_serverFixture.Url + "/generateJwtToken");
                 httpResponse.EnsureSuccessStatusCode();
-                var token = await httpResponse.Content.ReadAsStringAsync();
+                var token = httpResponse.Content.ReadAsStringAsync();
```

Comment:
What happened here?

Reply:
Well, the factory is returning the task it self, so it's not necessary to await it here.

Reply:
I see, can we change this test to move that logic all into the AccessTokenFactory. That'll be new coverage since everything else we have is pretty static.